wagtail/admin/site_summary.py

```diff
@@ -28,6 +28,7 @@ class PagesSummaryItem(SummaryItem):
 
     def get_context(self):
         root_page = get_explorable_root_page(self.request.user)
+        site_name = get_site_for_user(self.request.user)['site_name']
```

Comment:
@helenb Could we use `'root_page'` from the `get_site_for_user()` result here for the `root_page` variable? Right now, this view is triggering two calls to `get_explorable_root_page()`; one directly and another within `get_site_for_user()`

Reply:
Thanks @ababic - that is addressed in commit 587ec3a4238c852570fcd9dfd178c9e9e74a0769